### PR TITLE
Updated mod to 1.16.1

### PR DESCRIPTION
**Pull request by Xychic**
Just updated the mod to 1.16.1, shouldn't change any functionality

### CODE REVIEW

File: 1.16.1/.gitignore [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+.mtj.tmp/
+
+# Package Files #
+*.jar
+*.war
+*.ear
+
+# virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
+hs_err_pid*
+<<<<<<< HEAD
```
Comment (Domochevsky) [inline 1.16.1/.gitignore]: That's still here.

**Review by Domochevsky — COMMENTED** [review]
Again here: Looks pretty decent apart from indentation and that one line.

File: 1.16.1/src/main/java/net/wwsf/domochevsky/gentleharvest/GentleHarvest.java [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+    // Only does so if the block has reached max age
+	// Returns true if we succeeded and the event should be cancelled.
+	private boolean tryDropAndResetBlock(World world, BlockPos pos, IntegerProperty ageProperty, Block block, int fortune) {
+		BlockState blockState = world.getBlockState(pos);
+
+		if (blockState == null) { return false; }
+
+		int maxAge = Collections.max(ageProperty.getAllowedValues());
+		int age = blockState.get(ageProperty);
```
Comment (Domochevsky) [inline 1.16.1/src/main/java/net/wwsf/domochevsky/gentleharvest/GentleHarvest.java]: Oh yeah, while we're here: Can you add the bit of hardening mentioned in [this](https://github.com/Domochevsky/minecraft-gentleharvest/issues/2) issue, too?

File: 1.16.1/.gitignore (hunk 1 above)
Comment (Xychic) [reply 1.16.1/.gitignore]: Is the gitignore needed in the version specific folders?

File: 1.16.1/src/main/java/net/wwsf/domochevsky/gentleharvest/GentleHarvest.java (hunk 2 above)
Comment (Xychic) [reply 1.16.1/src/main/java/net/wwsf/domochevsky/gentleharvest/GentleHarvest.java]: I will get on that

File: 1.16.1/.gitignore (hunk 1 above)
Comment (Domochevsky) [reply 1.16.1/.gitignore]: Not in Minecraft, no, just as a git-related thing. Doesn't matter in that file in particular, but can lead to some odd behavior.

**Pull request merged**